diff --git a/ui/views/selection_controller.cc b/ui/views/selection_controller.cc
index b1a4514e7bdc4..3c66ab9e0cf4a 100644
--- a/ui/views/selection_controller.cc
+++ b/ui/views/selection_controller.cc
@@ -86,16 +86,6 @@ bool SelectionController::OnMousePressed(
     }
   }
 
-  if (handles_selection_clipboard_ && event.IsOnlyMiddleMouseButton() &&
-      !delegate_->IsReadOnly()) {
-    delegate_->OnBeforePointerAction();
-    const bool selection_changed =
-        render_text->MoveCursorToPoint(event.location(), false);
-    const bool text_changed = delegate_->PasteSelectionClipboard();
-    delegate_->OnAfterPointerAction(text_changed,
-                                    selection_changed | text_changed);
-  }
-
   return true;
 }
 
